css/index.css

```diff
@@ -143,4 +218,255 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+.middle-content1{
+    margin-bottom: 40px;
+}
+.text-container1{
+    display: flex;
+    flex-wrap: flex center;
+    vertical-align: center;
+}
+.paragraphs{
+	display: flex;
+	flex-direction: column;
+	align-content: center;
+	background: white;
+    margin: 5px;
+    padding-bottom: 100px;
+	height: 400px;
+    width: 600px;
+    border-bottom: 1px dashed black;
+}
+.paragraphs{
+	display: flex;
+}
+.p{
+	display: flex;
+	flex-direction: column;
+	align-content: center;
+	border: 1p transparent solid;
+	width: 400px;
+	margin-right: 55px;
+	padding: 0 10px;
+}
+.paragraphs button{
+    vertical-align: center;
+    border-radius: 5px;
+	height: 28px;
+	width: 120px;
+	padding: 5px 15px;
+	background: white;
+	border: 1px black solid;
+	cursor: pointer;
+	margin-top: 10px;
+}
+.paragraphs button:hover {
+	background: crimson;
+	color: white;
+}
+.blog-content{
```

Comment:
If anything, it might be a little cleaner if the background image was slightly opaque so that it didn't have such a large contrast

Reply:
https://guide.freecodecamp.org/css/background-opacity/